docs/reference/matcher_store.md

```diff
@@ -0,0 +1,9 @@
+# Datastore MatcherStore
+The `datastore.MatcherStore` interface defines all necessary persistence methods for `Libvuln`
+to provide its functionality.
+
+{{# godoc datastore.MatcherStore}}
```

Comment:
Should this just be referred to as the "Matcher Store" in prose? "Datastore MatcherStore" makes my eye twitch.